spec/jobs/recalculate_handover_date_job_spec.rb

```diff
@@ -77,7 +77,7 @@
     it 'raises an exception so the job will go into the retry queue' do
       expect {
         described_class.perform_now(offender_no)
-      }.to raise_error(Faraday::ClientError)
+      }.to raise_error(Faraday::ServerError)
```

Comment:
Technically this could be `Faraday::Error` to catch _any_ HTTP error.

Not necessarily needed, though – I think this is still clear as it is.